### PR TITLE
tests: Try a longer per-test timeout in unified_test

### PR DESCRIPTION
**Pull request by justinsb**
One minute may just be too close for comfort

### CODE REVIEW

File: tests/e2e/unified_test.go [hunk 1]
```diff
@@ -123,7 +125,9 @@ func testFixturesInSeries(ctx context.Context, t *testing.T, testPause bool, can
 
 	subtestTimeout := time.Hour
 	if targetGCP := os.Getenv("E2E_GCP_TARGET"); targetGCP == "mock" {
-		subtestTimeout = time.Minute
+		// We allow a two minutes for the test itself (particularly deep chains with retries),
+		// and kube-apiserver has a 1 minute timeout configured
+		subtestTimeout = 3 * time.Minute
```
Comment (acpana) [inline tests/e2e/unified_test.go:130]: I was looking at the timeout settings too at some point. FYI, it gets rebound here:

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/aa688ee381e87b732426429528bd754eed467e77/tests/e2e/unified_test.go#L124-L127

Comment (justinsb) [reply tests/e2e/unified_test.go:130]: I _think_ that's this line?

Comment (acpana) [reply tests/e2e/unified_test.go:130]: 🤦🏼 oh I thought I was reading the code in the harness

**Comment by acpana** [comment]
let's try it out!

/lgtm
/approve

happy to look again when not WIP. Maybe rebasing off of the latest origin will help with the existing mock timeouts too.

**Comment by acpana** [comment]
Let's try this out!

**Pull request merged**